website_blog_layout_options/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================================
+Layout Options and Extensions for Blog
+======================================
+
+
+* Adds 2 extra display types of posts to the post list of any blog:
+
+    * Teaser: The title of the post with a small teaser and a "read more" link.
+    * Complete: The complete post in the post list
```

Comment:
no truncation/limit? If not, maybe worth to list this as issue/improvement in roadmap?